xla/service/gpu/fusions/cudnn_test.cc

```diff
@@ -949,7 +949,7 @@ INSTANTIATE_TEST_SUITE_P(
                             HloOpcode::kNegate, HloOpcode::kRsqrt,
                             HloOpcode::kSin, HloOpcode::kSqrt, HloOpcode::kTan,
                             HloOpcode::kTanh}),
-                       ::testing::Values(5e-4)),
+                       ::testing::Values(1e-3)),
```

Comment:
Which of the ops will require the relaxed tolerance? And why is the tolerance worse on Blackwell?